README.md

```diff
@@ -156,11 +157,48 @@ turbo = auto
 ```
 
 ## How to run auto-cpufreq
-
-auto-cpufreq can be run by simply running the `auto-cpufreq` and following on screen instructions, i.e:
+Running just `auto-cpufreq` like so:
 
 `sudo auto-cpufreq`
 
+will give an error:
+
+`auto-cpufreq: wrong invocation. try --help for help.`
+
+auto-cpufreq should be run with with one of the following long options:
```

Comment:
`auto-cpufreq should be run with with one of the following long options:`

Let's just say:

`auto-cpufreq should be run with with one of the following options:`

Reply:
I like that you created this new section, let's add a line here for full description of each option please to [auto-cpufreq modes and options section](https://github.com/AdnanHodzic/auto-cpufreq/blob/1da39f0171594ff867f93206518e44ff837326ce/README.md#auto-cpufreq-modes-and-options).